packages/core/src/types.ts

```diff
@@ -20,14 +20,15 @@ type IPrismBaseConfig = {
   validateResponse: boolean;
   errors: boolean;
   upstreamProxy: string | undefined;
+  isProxy: boolean;
 };
 
 export type IPrismMockConfig = IPrismBaseConfig & {
   mock: object;
 };
 
 export type IPrismProxyConfig = IPrismBaseConfig & {
-  mock: false;
+  mock: object;
```

Comment:
any more specific type here?

Reply:
during our pair review / refactoring session, we set this to unknown